tests/Visual/Help.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+test('visual snapshot of help command output', function () {
+    $output = function () {
+        $process = (new Symfony\Component\Process\Process(['php', 'bin/pest', '--help', '--colors=never']));
+
+        $process->run();
+
+        return $process->getOutput();
+    };
+
+    expect($output())->toContain('Pest Options:');
```

Comment:
I would prefer this line to contain a snapshot. So we can see the entire thing. Sorry.

Reply:
No problem 👍🏻 Updated.